lib/smart_answer_flows/maternity-paternity-calculator/questions/earnings_for_pay_period.govspeak.erb

```diff
@@ -5,3 +5,7 @@
 <% content_for :hint do %>
   This is their earnings before deductions like PAYE, pension or National Insurance contributions.
 <% end %>
+
+<% content_for :error_message do %>
+  Use numbers like 12.34
```

Comment:
Has this error message been checked by a content designer? Only asking because the first commit explicitly stated the change was content designer approved, but this commit doesn't.

Reply:
Sorry, all of the content is provided by a CD.